src/dotnet/UI.Blazor/Components/Button/button.css

```diff
@@ -285,25 +279,33 @@ body.hoverable .btn.btn-round.btn-transparent:hover,
     @apply text-02;
 }
 body.hoverable .btn.btn-square.btn-transparent:hover::after,
-body.hoverable .btn.btn-round.btn-transparent:hover::after {
+.touch-capable .btn.btn-square.btn-transparent:active::after,
+body.hoverable .btn.btn-round.btn-transparent:hover::after,
+.touch-capable .btn.btn-round.btn-transparent:active::after {
     @apply hover-after;
-    @apply bg-transparent;
+    @apply bg-hover-after-primary
 }
 
 /* Button Square and Round Primary */
 body.hoverable .btn.btn-square.btn-primary:hover,
-body.hoverable .btn.btn-round.btn-primary:hover {
+.touch-capable .btn.btn-square.btn-primary:active,
+body.hoverable .btn.btn-round.btn-primary:hover,
+.touch-capable .btn.btn-round.btn-primary:active {
     @apply text-primary;
 }
 
 /* Button Square and Round Danger */
 body.hoverable .btn.btn-square.btn-danger:hover::after,
-body.hoverable .btn.btn-round.btn-danger:hover::after {
+.touch-capable .btn.btn-square.btn-danger:active::after,
+body.hoverable .btn.btn-round.btn-danger:hover::after,
+.touch-capable .btn.btn-round.btn-danger:active::after {
```

Comment:
Why do you use `body.hoverable` with body prefix and `.touch-capable` without body prefix while both are defined on body?